theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -466,6 +466,19 @@ def test_pool_grad(self):
             )
 
 
+# this has been a problem in the past
+def test_dnn_conv_border_mode():
```

Comment:
this test if failing travis with:

/home/travis/miniconda/envs/pyenv/lib/python2.6/site-packages/theano/sandbox/cuda/tests/test_dnn.py:476:5: F821 undefined name 'dnn_conv'

did you ran it? Did you confirm it raise the error before the fix?